docs/content.md

```diff
@@ -4,83 +4,79 @@
 
 ###  Global
 
-* Design Principles
-* Accessibility
-* Voice & tone (off link to Manual of Style?)
-* Designing for Safety
+* Accessibility: consider a11y in design early, list of tools
+* Design principles
+* Designing for safety
+* Content
 
-### Foundation
+### Foundations
 
-* Color: palette, use/brand (use more, use less), a11y (recommended combos)
-* Typography: scale, use (bold, normal, light, sans, mono) semantic markup/a11y,
+* Color: palette, use/brand (use more, use less), a11y (recommended combos), accent color
 * Spacing: scale, density on product vs spacious on marketing pages
+* Typography: scale, use (bold, normal, light, sans, mono) semantic markup/a11y
 
-### UI Patterns
+### Layout
+
+* Navigation
+* Responsive, desktop & mobile first
+* Templates: page layouts & hierarchy
 
+### UI patterns
+
+* Animation
+* Button usage
+* Defensive design: valley of safety vs destructive, destructive actions, danger
 * Empty states (blankslates)
-* Saving models
 * Loading states
-* Animation
-* Progressive disclosure (show more, expand etc.)
-* Defensive design (valley of safety vs destructive)
+* Messaging
+* Progressive disclosure: show more, expand, etc.
+* Search and filter
+* Tables and lists
 
-### Layout
+### Worflows
 
-* Page layouts & hierarchy
-* Responsive, desktop & mobile first
-* Navigation
-* Search
+* Creating
+* Feature onboarding
+* Saving
 
 ### Communication
 
+* Data formats
+* Emoji
+* Iconography
+* Imagery
+  * Octocats
+  * Illustrations
```

Comment:
Does this refer to technical illustrations like the images we use for "Feature preview" 

e.g.
![](https://user-images.githubusercontent.com/293280/79410958-85fbad00-7f56-11ea-9803-e893a6e72d57.png)

or to the octocat illustrations seen across the marketing pages and used in blankslates?

Reply:
TBH I left this section as it was before. From what I can see, this Imagery section is less UI assets and more decoration images, to simplify things a lot. Do you think we should restructure this proposal?